docs/contact-list.md

```diff
@@ -0,0 +1,28 @@
+# Security Contact List
+
+Beyond the initial fix team, a mailing list is setup to include additional
+organizations that can benefit from early CVE access. This mailing list
+is created per repository
+
+## OpenTelemetry Collector Distro Security Contacts List
+
+Currently on the opentelemetry-collector-distros list are representatives from:
+
+*
+*
+
+## Membership Criteria
```

Comment:
Can you please add that membership is decided by the sig-security for the moment? We'll use the criteria below as general guidelines, but we reserve the final decision is on us. 

Once we have practical experience with this process and confidence that the criteria make sense, we can remove ourselves as gatekeepers.